cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
@jkrocil You mean that the plain `return self._properties_form.properties_form` instead of the if to have always the same "code path" ?